packages/netlify-cms-core/src/components/Editor/EditorControlPane/Widget.js

```diff
@@ -275,13 +281,16 @@ export default class Widget extends Component {
       hasActiveStyle,
       editorControl,
       resolveWidget,
+      widget,
       getEditorComponents,
       query,
       queryHits,
       clearSearch,
       clearFieldErrors,
       isFetching,
       loadEntry,
+      isEditorComponent,
+      isNewEditorComponent,
```

Comment:
Separate concepts - sometimes a widget control behaves differently because it's an editor component, other times it needs to know if it was just inserted for things like autofocus.